packages/label-sync/src/label-sync.ts

```diff
@@ -156,23 +163,24 @@ async function reconcileLabels(github: GitHubAPI, owner: string, repo: string) {
     'kokoro: run',
     'question',
   ];
-  oldLabels.forEach(l => {
-    if (labelsToDelete.includes(l.name)) {
-      const p = github.issues
-        .deleteLabel({
-          name: l.name,
-          owner,
-          repo,
-        })
-        .then(() => {
-          console.log(`Deleted '${l.name}' from ${owner}/${repo}`);
-        })
-        .catch(e => {
-          console.error(`Error deleting label ${l.name} in ${owner}/${repo}`);
-          console.error(e.stack);
-        });
-      promises.push(p);
+  if (!Object.keys(oldLabels).length) {
+    for (const l of oldLabels) {
+      console.log(l.name);
```

Comment:
let's get rid of this label, but perhaps add something more descriptive right above `await github.issues`.

```
console.info(`deleting label ${l.name}`)
```

Reply:
Just deleted line 165, as it logs at the end if the label was deleted or not!